invokeai/configs/flux/flux1-schnell.yaml

```diff
@@ -0,0 +1,19 @@
+repo_id: "black-forest-labs/FLUX.1-schnell"
```

Comment:
Should we just keep these config values in memory rather than loading them from a YAML file?

Reply:
If we took this from an external source, I think it makes sense to keep it as `yaml` to make future comparisons w/ the source data very clear.

Reply:
They are copied from here https://github.com/black-forest-labs/flux/blob/main/src/flux/util.py#L26-L91. So, I'd be inclined to keep them as a python object.

Also, this comes back to another comment that I left, but we might just want to pip install the FLUX repo and then import from it.